<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QDialog" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="open">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reverse">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>反色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_grey">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>灰度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cool">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>冷色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_warm">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>暖色</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_saveas">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>另存为</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_soften">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>69</height>
            </size>
           </property>
           <property name="text">
            <string>柔化</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_old">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>老照片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_vague">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>模糊</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_sharpen">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>锐化</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
